### PR TITLE
[CORE-2143] Update readme

### PR DESCRIPTION
**Pull request by alexdicianu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,6 @@
 # Composer-enabled Upstream template
-## Experimental
 
 This is Pantheon's recommended starting point for forking new upstreams that work with the Platform's integrated
 Composer build process.
 
-Because it is under active development, you should not create permanent/production sites using this repository
-yet. We make no guarantees of backwards compatibility. Merging new updates from this repository may break existing
-sites.
+For more information and detailed installation guides, please visit the Integrated Composer Pantheon documentation: https://pantheon.io/docs/integrated-composer
```
Comment (alexdicianu) [inline README.md]: Docs article not published yet, see https://github.com/pantheon-systems/documentation/pull/5663

**Comment by alexdicianu** [comment]
+1

**Review by mbaynton — APPROVED** [review]

**Comment by mbaynton** [comment]
Suggest we hold on merging until the docs page is live

**Comment by alexdicianu** [comment]
docs are live https://pantheon.io/docs/integrated-composer

**Pull request merged**